data/src/main/java/com/teamagam/gimelgimel/data/layers/LayerFilenameSerializer.java

```diff
@@ -57,6 +53,27 @@ public VectorLayer toVectorLayer(String filename) {
                 "Unrecognized file (not a VectorLayer) was found: %s", filename));
     }
 
+    public String getFilePattern(VectorLayer vectorLayer) {
+        List<String> nameElements = getNameElements(vectorLayer);
+
+        nameElements.remove(SEVERITY_POSITION);
+        nameElements.add(SEVERITY_POSITION, ".*");
```

Comment:
I wonder whether it's better to check here for both severity types rather than using a wildcard. It causes me to feel uncomfortable, as if one could use it for an SQL injection or something :rofl: .
consider replacing ".*" with [ REGULAR | IMPORTANT ]